kubernetes/ingress.yml

```diff
@@ -1,10 +1,18 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: ingress-app
+  name: app-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /
```

Comment:
I thought we bailed on nginx

Reply:
well yes, never actually noticed it says nginx haha, but this is also just the default ingress template from kubernetes (see [this](https://kubernetes.io/docs/concepts/services-networking/ingress/)). and it works in this configuration